.github/workflows/publish.yaml

```diff
@@ -0,0 +1,29 @@
+# https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml
+name: Publish Package
+
+on:
+  release:
+    types: [published]
```

Comment:
This is a change in behavior, but seems OK: publish the package in PyPI when there is a published release on GitHub. We can test how this will work with tags and protected branches.